### ISSUE TITLE
Make UsageStatistics aware of the Zeebe tab

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#2115]
__What should we do?__

* [ ] Correctly attach metadata to diagram.opened events
* [ ] Collect deployment data for Zeebe diagrams

__Why should we do it?__

We want to integrate Zeebe into Camunda Modeler.

Child of https://github.com/bpmn-io/internal-docs/issues/216

**Comment by pinussilvestrus** [comment]
### Proposal: Data adjustments to existing events

_WIP_

**a Camunda BPMN diagram was opened**

```js
event: 'diagramOpened',
diagramType: 'bpmn',
diagramMetrics: { },
elementTemplateCount: 1,
elementTemplates: [ ],
engineProfile: {
  executionPlatform: 'Camunda Platform',
  executionPlatformVersion: '7.15.0'
} 
```

--> version could be lead to wrong data, since there is currently no way of setting it.

**a Camunda BPMN diagram was deployed**

```js
event: 'deployment',
diagramType: 'bpmn',
diagramMetrics: { },
deployment: { 
  context: 'deploymentTool',
  outcome: 'success'    
},
engineProfile: {
  executionPlatform: 'Camunda Platform',
  executionPlatformVersion: '7.15.0'
} 
```

**a Zeebe BPMN diagram was opened**

```js
event: 'diagramOpened',
diagramType: 'bpmn',
diagramMetrics: { },
engineProfile: {
  executionPlatform: 'Camunda Cloud',
  executionPlatformVersion: '1.0.0'
} 
```

--> note that no element templates data should be included
--> we could consider filling the `diagramMetrics` with the form data

```js
diagramMetrics: {
 tasks: {
    userTask: {
      count: 4,
      form: {
        count: 4,
         camundaForms: 4,
         other: 0
      }
    }
  }
}
```

**a Zeebe BPMN diagram was deployed**

```js
event: 'deployment',
diagramType: 'bpmn',
diagramMetrics: { },
deployment: { 
  context: 'deploymentTool',
  outcome: 'success',
  targetType: 'selfHosted | camundaCloud'
},
engineProfile: {
  executionPlatform: 'Camunda Cloud',
  executionPlatformVersion: '1.0.0'
} 
```

**Comment by nikku** [comment]
Good proposal. Makes it easy to filter by different platforms, too.

**Comment by pinussilvestrus** [comment]
The naming discussion (https://github.com/bpmn-io/internal-docs/issues/216) will potentially have an effect on the data shown here. I'd just go on creating the basic infrastructure, we can adjust the actual field names / possible field values later.

**Comment by pinussilvestrus** [comment]
Closed via #2123
